templates/vhost/locations/proxy.erb

```diff
@@ -1,3 +1,4 @@
+<% if @proxy -%>
     proxy_pass            <%= @proxy %>;
     proxy_read_timeout    <%= @proxy_read_timeout %>;
     proxy_connect_timeout <%= @proxy_connect_timeout %>;
```

Comment:
@ubellavance 
This PR is already merged, so no point adding a review / feedback here.
I don't think the issue is that `proxy_send_timeout` isn't part of the template; it's not a parameter for the location resource at all. As things in the past have tended to be added piecemeal as people needed them, there's not a lot of consistency in terms of which supported parameters are allowed in which contexts. As best I can tell, `proxy_send_timeout` isn't currently directly supported as a parameter for server blocks either; it's just supported at the top level nginx config. This could be because most people don't need to set different values for this for different sites, and / or that the people who do just use the `location_cfg_append` or similar directives.

That said, it's not a big limitation usually, because you can generally use the various custom directives to add those parameters if they're needed in server or location contexts. As @xaque208 says, PRs (with tests) are welcome.